### ISSUE TITLE
tf1 padding in conv is inconsistent with other versions

### ISSUE THREAD

**Issue by shachargluska** [osmr/imgclsmob#47]
tensorflowcv pads conv2d different than kerascv, tf2cv & gluoncv2.

On the keras/tf2 implementation of conv2d, the padding is as follows:

> padding: Int, or tuple of 2 ints, or tuple of 2 tuples of 2 ints.
If int: the same symmetric padding is applied to height and width.
If tuple of 2 ints: interpreted as two different symmetric padding values for height and width: (symmetric_height_pad, symmetric_width_pad).
If tuple of 2 tuples of 2 ints: interpreted as ((top_pad, bottom_pad), (left_pad, right_pad))

([source](https://www.tensorflow.org/api_docs/python/tf/keras/layers/ZeroPadding2D?version=stable))
When a tuple of 2 ints is passed, this is consistent with [mxnet version](https://beta.mxnet.io/api/gluon/_autogen/mxnet.gluon.nn.Conv2D.html).

Currently conv2d (for tf1) uses the following padding:
`paddings_tf = [[0, 0], list(padding), list(padding), [0, 0]]`
where `padding` is a pair of ints.
This is only consistent when `padding[0] == padding[1]`.
Most models keep H == W throughout, so this is okay.
But in networks such as squeezenext, this causes the tf1 version to behave differently.

**Comment by osmr** [comment]
This is an obvious bug. Thanks!
